tests/file/test_handle.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import unittest
+
+from azure.storage.file import (
+    FileService,
+)
+from tests.testcase import (
+    StorageTestCase,
+    record,
+    TestMode,
+)
+
+# ------------------------------------------------------------------------------
+TEST_SHARE_NAME = 'test'
+
+
+# ------------------------------------------------------------------------------
+
+class StorageHandleTest(StorageTestCase):
+    def setUp(self):
+        super(StorageHandleTest, self).setUp()
+        self.fs = self._create_storage_service(FileService, self.settings)
+
+    def tearDown(self):
+        return super(StorageHandleTest, self).tearDown()
+
+    def _validate_handles(self, handles):
```

Comment:
Could this take in the expected handle list, and compare that the actual results match, both in number of handles and handle contents?  I feel there's too much room for issues to slip in with tests that only verify the existence of parsed data, not the actual data contents

Reply:
It'd be possible if we could automate the set up of the tests. But right now it's manual (calling pause > bla.txt in powershell), so the exact response really depends on the status of the share, doing more validation would make the tests too brittle.